code/modules/mob/living/carbon/human/diona_gestalt.dm

```diff
@@ -222,6 +222,10 @@
 	set desc = "Allows you to detach specific nymph, and control it."
 	set category = "Abilities"
 
+	if(stat || paralysis || stunned || weakened || lying || restrained())
```

Comment:
we have `use_check_and_message` for this

Reply:
Handled, thank you.